dp/tools/skaffold_hooks/hooks.sh

```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-MAGMA_ROOT=$(realpath ..)
+MAGMA_ROOT=$(realpath .)
```

Comment:
Is it correct? (magma root should be root of git project, but here it looks like it's pointing to dp)

Reply:
I changed also context in skaffold build artifacts, and added dir attributes to hooks calls. So they will be called from the project root directory

Reply:
:+1: